virttest/qemu_vm.py

```diff
@@ -4127,23 +4127,20 @@ def block_stream(self, device, speed, base=None, correct=True):
         return self.monitor.block_stream(device, speed, base,
                                          cmd, correct=correct)
 
-    def block_mirror(self, device, target, speed, sync,
-                     format, mode="absolute-paths", correct=True):
+    def block_mirror(self, device, target, sync,
+                     correct=True, **kwargs):
         """
         Mirror block device to target file;
 
         :param device: device ID
         :param target: destination image file name;
-        :param speed: max limited speed, default unit is B/s;
         :param sync: what parts of the disk image should be copied to the
                      destination;
-        :param mode: new image open mode
-        :param format: target image format
         :param correct: auto correct cmd, correct by default
         """
         cmd = self.params.get("block_mirror_cmd", "drive-mirror")
```

Comment:
The default should be 'drive-mirror'.
It's not your fault.
But I think you can fix it.